helmfile.d/10-services.yaml

```diff
@@ -288,62 +302,64 @@ releases:
         value: {{ .Values.server_name }}
 
   - name: kratos
```

Comment:
It's good that name of the chart and name of the release to match, otherwise during deployment you'll have a pod named `radar-kratos-kratos` which unnecessarily long. If both chart name and release name match the pod will named something like `radar-kratos`